### PR TITLE
[Feature] : Add improvement in the art section

### PR DESCRIPTION
**Pull request by Himanshushakyaa**
## Description

-   [Ticket Link](#) (if any)
-   Provide a small description ( 2 - 3 lines) of the Pull Request

## UI screenshot or Video

-   Please include a screenshot or video to help reviewers understand the specific part of the application being modified. Multiple screenshots are appreciated

---

### General:

-   [ ] **PR Size :** Confirming PR size < 20 files or pre-approved if > 20 files.
-   [ ] **Self Review :** Confirming I have self-reviewed my PR
-   [ ] **Peers Review :** - [ ] @Dev 1
-   [ ] Confirming I have added all sensitive information-related should be inside .env files

### Client:

-   [ ] **Console :** Confirming no console errors in the UI for this update.

-   [ ] **Import order :** Confirming correct import order.

```
# External Import
import React, { useState, useEffect } from 'react';
import { useParams } from 'react-router-dom';
import { useDispatch, useSelector, shallowEqual } from 'react-redux';

# Internal Import
import { } from '../../Api/ApiRoutes';
import { } from '../../constants/AppConstant';
```

-   [ ] **Re-renders :** Confirming minimal re-renders for new React components.

-   [ ] **Network calls :** Confirming efficient network/API calls, minimized as needed.

-   [ ] **Util function comments :** Confirming comprehensive comments for utility functions.

-   [ ] **Image optimization :** Confirming I have checked that all images are optimized. If not, please use https://tinyjpg.com/ or https://vecta.io/nano.

-   [ ] **Dead code :** Confirming I have removed all comments code which is of no use OR written why I have kept the commented code.

### Confirming different viewport testing of the entire flow:

-   [ ] Tested on 280px viewport
-   [ ] Tested on 320px viewport
-   [ ] Tested on 375px viewport
-   [ ] Tested on 425px viewport
-   [ ] Tested on 768px viewport
-   [ ] Tested on 1024px viewport
-   [ ] Tested on 1440px viewport
-   [ ] Tested on 2560px viewport

### End-to-end testing:

-   [ ] **Happy flow :** Successfully tested end-to-end happy flow, all working as expected.
-   [ ] **Error flow :** Tested UI flow with error scenarios, gracefully handled errors on UI
-   [ ] **Common component changed :** Tested change in common component across views to prevent breaking elsewhere.

### CODE REVIEW

File: src/assets/images/the-art-poster.png
Comment (ram-primathon) [inline src/assets/images/the-art-poster.png:1]: What is this? 1MB image? why?

**Review by ram-primathon — CHANGES_REQUESTED** [review]

File: src/assets/images/the-art-poster.png
Comment (Himanshushakyaa) [reply src/assets/images/the-art-poster.png:1]: sir iye 6 mb ki image thi tinify use krne k baad iye 1 mb  ki  huyi isse km nhi ho rhi thi

Comment (ram-primathon) [reply src/assets/images/the-art-poster.png:1]: Because you have taken  W: 3152px | H: 1433px Image. earlier it was small please use the same old image

**Comment by ram-primathon** [comment]
Please fix the image issue in your next PR

**Pull request merged**